.travis.yml

```diff
@@ -19,7 +19,9 @@ before_script:
 install:
   - if [[ $DB == mysql ]]; then pip install -qU mysqlclient; fi
   - if [[ $DB == postgres ]]; then pip install -qU psycopg2; fi
-  - pip install -qU $DJANGO
+  - pip install -qU $DJANGO coveralls
```

Comment:
IMHO, it's better if we leave it outside test_requirements because it's not required to run tests. Only required when we want a coverage run (tox -e coverage)